BasicSample/ProjectSettings/ProjectSettings.asset

```diff
@@ -13,7 +13,7 @@ PlayerSettings:
   useOnDemandResources: 0
   accelerometerFrequency: 60
   companyName: DefaultCompany
```

Comment:
Do we want to change this to Microsoft?

Reply:
We could. I remember some concerns about people using the samples as a starting point for building their own apps, but not updating this companyName field. Although, it seems like it would be difficult to rename the app without noticing this field, so we might be safe to put Microsoft here.

Reply:
I looked at some related Microsoft samples, and they also had "Microsoft" set in this companyName field. I'll update this here too.

Reply:
Updated it in the latest commit